### PR TITLE
 #157732734 set up postgreSQL

### PR DESCRIPTION
**Pull request by OKiMaureen**
#### What does this PR do?
set up postgreSQL
 #### Description of Task to be completed?
- to enable the app connect to database
#### What are the relevant pivotal tracker stories?
-  #157732734
#### Questions:
-

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# DATABASE
```
Comment (paradisekelechi) [inline .env]: Your .env file should not be committed and pushed

File: server/config/config-test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const config = {
+  database: process.env.DATABASE_TEST_URL,
+};
+module.exports = config;
```
Comment (paradisekelechi) [inline server/config/config-test.js]: Why the use of module.exports ?

File: server/config/config.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const config = {
+
+  database: process.env.DATABASE_URL,
+
+};
+module.exports = config;
```
Comment (paradisekelechi) [inline server/config/config.js]: Use export default

Comment (paradisekelechi) [reply server/config/config.js]: This file is a replica of config-test.js . . . . Kindly refactor this. Remember DRY (Don'r Repeat Yourself!)

File: server/models/index.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import configDev from '../config/config';
+import queries from './database';
+
+
+let config;
+if (process.env.NODE_ENV === 'development') {
+  config = configDev;
+} else {
+  config = configTest;
+}
```
Comment (paradisekelechi) [inline server/models/index.js:12]: What about production environment?

File: .env (hunk 1 above)
Comment (OKiMaureen) [reply .env]: ok, i'll rectify that

File: server/config/config-test.js (hunk 2 above)
Comment (OKiMaureen) [reply server/config/config-test.js]: ok

File: server/config/config.js (hunk 3 above)
Comment (OKiMaureen) [reply server/config/config.js]: ok, i will do that

**Pull request merged**